signer/src/bip44.rs

```diff
@@ -10,14 +10,14 @@ use zeroize::Zeroize;
 
 const HARDENED_BIT: u32 = 1 << 31;
```

Comment:
mm... I see two things here...
1. Do we still need this here? 
2. probably ledger-rs is not the best place for the bip44 crate... We can discuss this.. we probably need a new independent repo for these things.

Reply:
The `bip44` crate in `ledger-rs` can replace this. This should not be needed anymore.

I agree `bip44` should be its own repo.